src/pykeen/lr_schedulers/__init__.py

```diff
@@ -34,11 +34,11 @@
     "StepLR",
 ]
 
-#: A wrapper around the hidden scheduler base class
-LRScheduler = _LRScheduler
+# fixme: bring this upstream to class_resolver.contrib?
+lr_scheduler_resolver = ClassResolver.from_subclasses(LRScheduler, default=ExponentialLR, suffix="LR")
```

Comment:
@cthoyt this could be fixed in `class-resolver` upstream; we could also think about adding some logic to make this work in `torch>=2.0` *and* `torch < 2`, though I am not sure whether to do that.

Reply:
i like the idea of adding more contrib to class-resolver. If it's no so hard to make compatibility, then great, otherwise we should aim for only supporting the latest version